### ISSUE TITLE
ImportError:...unicodedata.pyd

### ISSUE THREAD

**Issue by az0** [bleachbit/bleachbit#761]
[Build 1495](https://ci.appveyor.com/project/az0/bleachbit/builds/30362594)
BleachBit-3.1.0.1495-portable

Windows 10 64-bit

This happens after I use the Delete or Preview buttons

```
Traceback (most recent call last):
  File "C:\projects\bleachbit\bleachbit\GUI.py", line 655, in run_operations
  File "C:\projects\bleachbit\bleachbit\GUI.py", line 661, in preview_or_run_operations
  File "C:\projects\bleachbit\bleachbit\Worker.py", line 25, in <module>
  File "C:\projects\bleachbit\bleachbit\DeepScan.py", line 30, in <module>
  File "<loader>", line 10, in <module>
  File "<loader>", line 8, in __load
ImportError: (DLL load failed: No se puede encontrar el módulo especificado.) 'C:\\Users\\username\\Downloads\\BleachBit-3.1.0.1495-portable\\BleachBit-Portable\\unicodedata.pyd'
```

**Comment by mkhon** [comment]
I can't reproduce this. Windows 10 x64

**Comment by az0** [comment]
I wonder whether you didn't see it on your system because you have the development environment or if you had the setting disabled to check for updates.

In either case, it looks fixed in ab76ebd39616e23e34036b8c1f5602309326b40b
